completion/available/npm.completion.bash

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 # npm (Node Package Manager) completion
+# https://docs.npmjs.com/cli/completion
 
 if command -v npm &>/dev/null
 then
-  eval "$(npm completion)"
+  source <(npm completion)
```

Comment:
I took it from here : https://docs.npmjs.com/cli/completion
It works on my system [GNU bash, version 4.3.46(1)-release (x86_64-pc-linux-gnu)].

Did the previous `eval "$(npm completion)"` worked alright? I can change it back.